### DISCUSSION TITLE
add a documentation for the handleMessageBatch partial ack

### DISCUSSION THREAD

**Discussion by ruddenchaux** [bbc/sqs-consumer#461] [Help] [answered]
### Summary

It can be useful to have documentation explaining the feature added with #255.

**Comment by nicholasgriffintn** [comment, marked as answer]
Hey sorry for the delayed response, we do have some documentation in the Usage section of the readme here: https://github.com/bbc/sqs-consumer?tab=readme-ov-file#usage

Outside of that, the comments also document the usage: https://bbc.github.io/sqs-consumer/interfaces/ConsumerOptions.html#handleMessage

What particular ways do you think we could improve here? Is there anything in particular that you found undocumented?

**Comment by ruddenchaux** [comment]
Hi, you are right. I was referring to partials ack with the batch operations, and I see documentation present on  https://bbc.github.io/sqs-consumer/interfaces/ConsumerOptions.html#handleMessage.

> In the case that you need to ack only some of the messages, return an array with the successful messages only.

Thank you

**Reply by nicholasgriffintn** [reply]
Awesome! Glad to hear it's sorted.
